src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1005,6 +1008,9 @@
   - name: Enable open-service-mesh addon.
     text: az aks enable-addons --name MyManagedCluster --resource-group MyResourceGroup --addons open-service-mesh
     crafted: true
+  - name: Enable azure-defender addon with workspace resourceId.
+    text: az aks enable-addons --name MyManagedCluster --resource-group MyResourceGroup --addons azure-defender --appgw-subnet-cidr 10.2.0.0/16 --workspace-resource-id WorkspaceResourceId
```

Comment:
Was --appgw-subnet-cidr 10.2.0.0/16  left in by accident or will this only work with app gateway?

Reply:
By accident. Removed in latest commit